package.json

```diff
@@ -4,12 +4,20 @@
   "license": "MIT",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "default": "./dist/index.js"
+    },
+    "./package.json": "./package.json"
```

Comment:
I don't think we need to do this in our packages anymore. It was a result of how our bundler plugins worked, but they've since been fixed

Reply:
better safe than sorry - I'm not sure if rollup-plugin-svelte and svelte-loader can handle it yet

Reply:
I'm pretty sure we updated them. Doesn't matter much, but figured I'd let you know